{"package_name": "java.security.cert", "subclass_list": [], "class_description": "Certificate Encoding Exception. This is thrown whenever an error\n occurs while attempting to encode a certificate. ", "class_type": "class", "typealias": [], "interface_list": [""], "class_inherit_list": ["CertificateException"], "Methods": [{"params": [], "class_name": "CertificateEncodingException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Constructs a CertificateEncodingException with no detail message. A\n detail message is a String that describes this particular\n exception.\n", "method_declar": ""}, {"params": [{"param_name": ["message"], "param_type": "String", "param_description": " the detail message."}], "class_name": "CertificateEncodingException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Constructs a CertificateEncodingException with the specified detail\n message. A detail message is a String that describes this\n particular exception.\n", "method_declar": ""}, {"params": [{"param_name": ["message"], "param_type": "String", "param_description": " the detail message (which is saved for later retrieval\n        by the Throwable.getMessage() method)."}, {"param_name": ["cause"], "param_type": "Throwable", "param_description": " the cause (which is saved for later retrieval by the\n        Throwable.getCause() method).  (A null value is permitted,\n        and indicates that the cause is nonexistent or unknown.)"}], "class_name": "CertificateEncodingException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Creates a CertificateEncodingException with the specified\n detail message and cause.\n", "method_declar": ""}, {"params": [{"param_name": ["cause"], "param_type": "Throwable", "param_description": " the cause (which is saved for later retrieval by the\n        Throwable.getCause() method).  (A null value is permitted,\n        and indicates that the cause is nonexistent or unknown.)"}], "class_name": "CertificateEncodingException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Creates a CertificateEncodingException\n with the specified cause and a detail message of\n (cause==null ? null : cause.toString())\n (which typically contains the class and detail message of\n cause).\n", "method_declar": ""}], "class_name": "CertificateEncodingException", "Vars": null}